.github/workflows/scanner-offline-bundle-update.yaml

```diff
@@ -1,37 +1,48 @@
 name: Scanner release offline vulnerability bundle update
+
 on:
   schedule:
     - cron: '0 */3 * * *'
+  pull_request:
+    types:
+    - opened
+    - reopened
+    - synchronize
   workflow_dispatch:
 
 jobs:
-  read-release-versions:
+  generate-version-matrix:
```

Comment:
can you add the word `offline` to each job name? Otherwise, when looking at the CI jobs that ran for this PR, it wasn't immediately clear these were for offline. Perhaps even add scanner somewhere, too

Reply:
be872c3370